appveyor.yml

```diff
@@ -30,16 +31,18 @@ environment:
     C:\\Program Files\\Git\\cmd;\
     C:\\Program Files\\Git\\usr\\bin;\
     C:\\Program Files\\AppVeyor\\BuildAgent;"
+
   CC: "clang-cl.exe"
   CXX: "clang-cl.exe"
-  LINKER: "lld-link.exe"
 
   VCVARSALL_PATH: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build
 
   matrix:
     - TARGET: i686-pc-windows-msvc
       CROSS_COMPILE: 1
     - TARGET: x86_64-pc-windows-msvc
+    - TARGET: x86_64-pc-windows-msvc
+      FEATURES: --features debugmozjs
```

Comment:
The reason we didn't have this job before is because of https://github.com/servo/mozjs/issues/189.

Reply:
Yeah I figured that out but I wanted to see if a debug build would tell me more about the `enumerate` test failing. Maybe I fumbled updating the C++ glue and I badly copied a prototype somewhere.